shardingsphere-features/shardingsphere-authority/shardingsphere-authority-common/src/main/java/org/apache/shardingsphere/authority/provider/natived/builder/dialect/PostgreSQLPrivilegeHandler.java

```diff
@@ -130,19 +160,19 @@ private void fillRolePrivileges(final Map<ShardingSphereUser, NativePrivileges>
             }
         }
     }
-    
+
     private void fillRolePrivileges(final Map<ShardingSphereUser, NativePrivileges> userPrivilegeMap, final ResultSet resultSet) throws SQLException {
         Optional<ShardingSphereUser> user = findShardingSphereUser(userPrivilegeMap, resultSet);
         if (user.isPresent()) {
             userPrivilegeMap.get(user.get()).getAdministrativePrivileges().getPrivileges().addAll(loadRolePrivileges(resultSet));
         }
     }
-    
+
```

Comment:
Please give a look at these blank space changes.

Reply:
This should be a different specification. I used idea’s automatic formatting. In his specification, blank lines do not contain any characters, but in our project, blank lines seem to contain a Tab placeholder. Which one should we choose? Which way do you think is better?

Reply:
Hi, it is suggested to indent each line.
BTW, if you like, [IDEA Settings](https://shardingsphere.apache.org/community/cn/contribute/code-conduct/) is convenient to import into IDEA.

Reply:
Okay, thank you, I'm happy to do this.